composer.json

```diff
@@ -8,7 +8,7 @@
         "php": ">=5.5.9",
         "anahkiasen/arrounded": "^0.6",
         "bugsnag/bugsnag-laravel": "^1.4",
-        "laravel/framework": "5.1.*",
+        "laravel/framework": "^5.1",
```

Comment:
Because of the lock file? Didn't `composer update laravel/framework` update the lock file?